pkg/debug/checkups/power_windows.go

```diff
@@ -0,0 +1,64 @@
+//go:build windows
+// +build windows
+
+package checkups
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/kolide/launcher/pkg/agent"
+)
+
+type powerCheckup struct{}
+
+func (p *powerCheckup) Name() string {
+	return "Power Report"
```

Comment:
If this is destined to only be windows, I think we should include that in the name.

Reply:
I thought maybe we'd want to pull something similar for macos/linux in the future? And we probably don't need to include the OS for the device in the checkup name